hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/hooks/useNewFields.tsx

```diff
@@ -127,20 +131,23 @@ export const useNewFields = (
             new_parent: {
                 label: formatMessage(MESSAGES.parent),
                 order: 3,
+                fieldType: '',
                 formatValue: val => (
                     <LinkToOrgUnit orgUnit={val as ShortOrgUnit} />
                 ),
             },
             new_groups: {
                 label: formatMessage(MESSAGES.groups),
                 order: 4,
+                fieldType: 'array',
                 formatValue: val => (
                     <span>{getGroupsValue(val as NestedGroup[])}</span>
                 ),
             },
             new_location: {
                 label: formatMessage(MESSAGES.location),
                 order: 5,
+                fieldType: '',
```

Comment:
we can use string for this no ?

Reply:
I don't think it would be possible to use a string for that as the **new location** field is structured like: 

```
"new_location": {
        "altitude": 0,
        "latitude": -11.6861,
        "longitude": 7.39085
}
```